main.tf

```diff
@@ -37,3 +39,75 @@ resource "google_logging_project_sink" "sink" {
 
   depends_on = [var.module_depends_on]
 }
+
+resource "google_logging_organization_sink" "sink" {
+  count = var.organization != null ? 1 : 0
```

Comment:
we should make it conflicting for now... priority can be:

- project != null -> create project sink
- project == null && folder != null -> folder sink
- project == null && folder == null && org != null -> org
- all == null -> project with default from provider..

if all are non null we will just create the project and ignore all others..

we can document this with all variables conflict with each other.... there is no validation for this atm... still would love to have an assertion provider for this hehe..